TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/ArmSystem.java

```diff
@@ -174,78 +145,78 @@ private void goHome () {
                 homing = false; // We're done!
             }
         }
-        updateHeight(1);
+        raise(1);
 
     }
 
-    private void openGripper () {
-        gripper.setPosition(GRIPPER_OPEN);
+    private void openGripper() {
+        servoEnumMap.get(ServoNames.GRIPPER).setPosition(GRIPPER_OPEN);
     }
 
-    private void closeGripper () {
-        gripper.setPosition(GRIPPER_CLOSE);
+    private void closeGripper() {
+        servoEnumMap.get(ServoNames.GRIPPER).setPosition(GRIPPER_CLOSE);
     }
 
-    public void toggleGripper () {
-        if (Math.abs(gripper.getPosition() - GRIPPER_CLOSE)
-                < Math.abs(gripper.getPosition() - GRIPPER_OPEN)) {
+    public void toggleGripper() {
+        if (Math.abs(servoEnumMap.get(ServoNames.GRIPPER).getPosition() - GRIPPER_CLOSE)
```

Comment:
My general rule of thumb is that if a method is called twice, set it to a variable.